test/indexer/test_indexer_controller.py

```diff
@@ -0,0 +1,139 @@
+import json
+import logging
+
+import boto3
+import mock
+from moto import (
+    mock_sqs,
+    mock_sts,
+)
+
+from azul import (
+    config,
+    queues,
+)
+from azul.azulclient import (
+    AzulClient,
+)
+from azul.indexer import (
+    BundleFQID,
+)
+from azul.indexer.index_controller import (
+    IndexController,
+)
+from indexer import (
+    ForcedRefreshIndexService,
+    IndexerTestCase,
+)
+
+log = logging.getLogger(__name__)
+
+
+class Record:
```

Comment:
Any reason we can't use SQSRecord from CHalice?

Reply:
was easier when prototyping to mock this functionality out. refactored to SQSRecord.